tests/unit_tests/utils/test_general.py

```diff
@@ -455,28 +312,26 @@ def test_get_log_excerpt(tmp_test_directory) -> None:
     )
 
 
-def test_file_has_text(tmp_test_directory, caplog) -> None:
+def test_file_has_text(tmp_test_directory, caplog, file_has_text) -> None:
```

Comment:
This is now a function in conf.py. We usually don't test these functions, but I think it is good to keep it (just to note that we don't test general.py functions here). All good!